browser/safari/find_darwin.go

```diff
@@ -7,10 +7,29 @@ import (
 	"path/filepath"
 )
 
+func fileExists(filename string) bool {
```

Comment:
Can you remove this file check? I think late failing when trying to open the file is better. the automatic search FindAllCookieStores will probably will deliver a few CookieStores. Then it can be filtered and only the remaining ones should be checked (simply opened). Oherwise profiles stored on the network will slow down.

Reply:
Maybe you need to see another pr, which he achieved.